### PR TITLE
FossGraphView refactoring

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Moved vulnerability badge to separate `FC` - `vulnerabilityBadge`
 * Improved `vulnerabilityBadge`
 * Fixed `tableComponent` visuals
 * Slightly improved `fossGraph` visuals

### TODO:
 * Fix ugly comments
 * Add card layout to `fossGraph`

### How it looks:

https://github.com/saveourtool/save-cloud/assets/35039155/f524cd30-6ce2-4412-b764-3172c8bdf357

### CODE REVIEW

**Comment by kgevorkyan** [comment]
could be the pagination menu finally hidden? if there are less than 10 records?

**Comment by sanyavertolet** [comment]
> could be the pagination menu finally hidden? if there are less than 10 records?

will do it now

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt [hunk 1]
```diff
@@ -179,10 +181,10 @@ val vulnerabilityInfoTab: FC<VulnerabilityInfoTabProps> = FC { props ->
 
     div {
         className = ClassName("d-flex justify-content-end")
-        buttonBuilder("Add project", classes = "mr-2", isOutline = true) {
+        buttonBuilder(faPlus, classes = "mr-2", isOutline = true) {
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt:184]: may be add description on hover?

**Review by kgevorkyan — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt:184]: Will do it

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt]: Done

**Review by Cheshiriks — APPROVED** [review]

**Comment by kgevorkyan** [comment]
> > could be the pagination menu finally hidden? if there are less than 10 records?
> 
> Now if there is only one page, no paging info is shown

finally

**Pull request merged**